centaur/src/main/resources/standardTestCases/lots_of_inputs_papiv2.test

```diff
@@ -0,0 +1,19 @@
+# This test makes sure that:
+# - 10000 output files are all found and collected by the glob() method
+# - 10000 input files to a task doesn't make anything explode inappropriately
+name: lots_of_inputs_papiv2
+testFormat: workflowsuccess
+tags: [ big_metadata ]
+backends: [ Papiv2 ]
```

Comment:
I think the preference now is to use the `-e` flags to decide which backends run which tests